sphinxcontrib/openapi/directive.py

```diff
@@ -67,6 +67,7 @@ class OpenApi(Directive):
     option_spec = {
         'encoding': directives.encoding,    # useful for non-ascii cases :)
         'paths': lambda s: s.split(),       # endpoints to be rendered
+        'request': directives.flag,         # print the request body structure
```

Comment:
FYI, I'm going to revisit option names before next release. So please, do not think that your feature will be thrown away. It may be got renamed.

Reply:
No problems with it. Looking forward to start using the updates.
Thank you.